agent/agent-tooling/src/test/resources/ApplicationInsights_SpanProcessor.json

```diff
@@ -85,6 +85,22 @@
         },
         "id": "log/extractAttributes"
       },
+      {
+        "type": "log",
+        "include": {
+          "matchType": "regexp",
+          "logBodies": [
+            ".*password.*"
+          ]
+        },
+        "name": {
+          "fromAttributes": [
+            "loggerName"
```

Comment:
this doesn't look like the otel attribute name, so I'm surprised this is working?

Reply:
loggerName is different from logName.. i changed logName to logBodies for matching a few lines above..

Reply:
let's chat about this and the one above.